Packages/Sites/Sfi.Kateheo/Resources/Private/Templates/NodeTypes/News.html

```diff
@@ -30,6 +31,17 @@ <h3 itemprop="headline">{neos:contentElement.editable(property: 'title', tag: 's
 							</neos:link.node>
 						</f:if>
 					</f:for>
+					<!-- tags -->
+					<f:for each="{tags}" as="item">
+						<f:if condition="{item.properties.title}">
+							<i class="fa fa-tag"></i>
+							<neos:link.node node="{item}">
+								<span class="news-list-tag" itemprop="tag" itemscope="itemscope" itemtype="http://schema.org/Person">
```

Comment:
Yes, either or. It would be interesting to explore this topic of microdata more deeply, so if you're up to it, it's cool. When you are done, you could also add it to sfi.ru...